apps/web/package.json

```diff
@@ -113,6 +113,7 @@
     "zod": "^3.8.2"
   },
   "devDependencies": {
+    "uglify-js": "latest",
```

Comment:
what is this dependency?

Reply:
This is to uglify the JS string for theme code.

Reply:
can we get away without a new dependency?

Reply:
Well technically we can, as the size of the theme code isn't too big right now, so improvement through minification won't be much. But it's good to have minification already in place so that the payload served is always to minimum.

What is the motivation to avoid it? Apart from a small increase installation time of deps

Reply:
@zomars Thanks. Fixed that. Though it was still working, it makes logical sense to keep it in dependencies.

@PeerRich @zomars Do you guys still feel that uglify-js should be removed?

Reply:
i still don't quite understand what this dependency is for. can we not do that with a simple one liner function?

Reply:
in general, we try to avoid unnecessary packages, especially if they are only being used once

Reply:
The purpose is to uglify the `applyThemeAndAddListener`function which is small right now.

Let me remove uglify-js till we have stronger reason to add the dependency. The result would be that the function would be sent as is to the client now.

Reply:
@PeerRich uglify-js is now removed.